File: eval/fixtures/legacy-directory-backed-jaxws-commerce/src/main/java/com/acme/legacy/orders/OrderFulfillmentRepository.java
package com.acme.legacy.orders;

public class OrderFulfillmentRepository {
  public FulfillmentRepriceResult writeFulfillmentReprice(String orderId) {
    String sql = "SELECT * FROM orders.jaxws_order_fulfillment_backlog WHERE order_id = ?";
    return new FulfillmentRepriceResult(sql);
  }
}

File: eval/fixtures/legacy-directory-backed-asmx-commerce/src/Legacy/Orders/OrderFulfillmentRepository.cs
namespace Acme.Legacy.Orders;

public class OrderFulfillmentRepository
{
  public string WriteFulfillmentReprice(string orderId)
  {
    string sql = "SELECT * FROM orders.asmx_order_fulfillment_backlog WHERE order_id = @orderId";
    return sql;
  }
}
